pkg/cmd/server/server_test.go

```diff
@@ -97,3 +98,39 @@ func TestArkResourcesExist(t *testing.T) {
 	arkAPIResourceList.APIResources = arkAPIResourceList.APIResources[:3]
 	assert.Error(t, server.arkResourcesExist())
 }
+
+func TestDefaultVolumeSnapshotLocations(t *testing.T) {
+	namespace := "heptio-ark"
+	arkClient := fakeclientset.NewSimpleClientset()
+	location := &v1.VolumeSnapshotLocation{
```

Comment:
would be nice to add a couple more cases to cover multiple providers/multiple VSLs per provider :)